core/src/main/java/org/infinispan/health/HealthStatus.java

```diff
@@ -0,0 +1,24 @@
+package org.infinispan.health;
+
+/**
+ * General Health status.
+ *
+ * @author Sebastian Łaskawiec
+ * @since 9.0
+ */
+public enum HealthStatus {
+    /**
+     * Given entity is unhealthy.
```

Comment:
Unhealthy means that some partitions are in `DEGRADED_MODE`. I added more information to the javadoc. Thanks for pointing this out.